models/fates/R/write.configs.FATES.R

```diff
@@ -202,143 +213,94 @@ write.config.FATES <- function(defaults, trait.values, settings, run.id){
-     pft.name <- names(trait.values)[i]
-     if(is.null(pft.name) | is.na(pft.name)){
-       PEcAn.logger::logger.error("pft.name missing")
-     } else {
-       PEcAn.logger::logger.info(paste("PFT =",pft.name))
-       PEcAn.logger::logger.debug(paste0("fates-clm PFT number: ",which(pftnames==pft.name)))
-     }
-     if(pft.name == 'env') next   ## HACK, need to remove env from default
+   #pftnames <- stringr::str_trim(tolower(ncdf4::ncvar_get(clm.param.nc,"pftname")))
+   #PEcAn.logger::logger.debug(paste0("CLM PFT names: "),pftnames)
+   #for (i in seq_len(npft)) {
+     #pft <- trait.values[[i]]
+     #print(c("PFT",i))
+     #PEcAn.logger::logger.info(pft)
+     #pft.name <- names(trait.values)[i]
+     #if(is.null(pft.name) | is.na(pft.name)){
+       #PEcAn.logger::logger.error("pft.name missing")
+     #} else {
+       #PEcAn.logger::logger.info(paste("PFT =",pft.name))
+       #PEcAn.logger::logger.debug(paste0("fates-clm PFT number: ",which(pftnames==pft.name)))
+     #}
+     #if(pft.name == 'env') next   ## HACK, need to remove env from default
      
      ## Match PFT name to COLUMN
-     ipft <- match(tolower(pft.name),pftnames)
-     PEcAn.logger::logger.debug(paste0("ipft: ",ipft))
+     #ipft <- match(tolower(pft.name),pftnames)
+     #PEcAn.logger::logger.debug(paste0("ipft: ",ipft))
                                       
-     if(is.na(ipft)){
-       PEcAn.logger::logger.severe(paste("Unmatched PFT",pft.name,
-                          "in FATES. PEcAn does not yet support non-default PFTs for this model"))
-     }
+     #if(is.na(ipft)){
+       #PEcAn.logger::logger.severe(paste("Unmatched PFT",pft.name,
+                          #"in FATES. PEcAn does not yet support non-default PFTs for this model"))
+     #}
      
      # hard code hack until we can use more than 2 pfts in FATES 
-     ipft <- 2
-     PEcAn.logger::logger.debug(paste0("*** PFT number hard-coded to ", ipft," in fates. This will be updated when FATES allows more PFTs"))
+     #ipft <- 2
+     #PEcAn.logger::logger.debug(paste0("*** PFT number hard-coded to ", ipft," in fates. This will be updated when FATES allows more PFTs"))
      
      ## Special variables used in conversions
 #     leafC <- pft['leafC']/100  ## percent to proportion
-     leafC <- NA
-     if(is.na(leafC)) leafC <- 0.48
+     #leafC <- NA
+     #if(is.na(leafC)) leafC <- 0.48
      
      # determine photo pathway
-     photo_flag <- ncdf4::ncvar_get(fates.param.nc,varid="fates_c3psn", start = ipft, count = 1)
-     PEcAn.logger::logger.debug(paste0("Photosynthesis pathway flag value: ", photo_flag))
+     #photo_flag <- ncdf4::ncvar_get(fates.param.nc,varid="fates_c3psn", start = ipft, count = 1)
+     #PEcAn.logger::logger.debug(paste0("Photosynthesis pathway flag value: ", photo_flag))
      
      ## Loop over VARIABLES
-     for (v in seq_along(pft)) {
-       var <- names(pft)[v]
+     #for (v in seq_along(pft)) {
+       #var <- names(pft)[v]
```

Comment:
Similar to previous comment, what's the reason for commenting out the last ~50 lines?